diff --git a/node_modules/@orama/plugin-docusaurus-v3/dist/index.js b/node_modules/@orama/plugin-docusaurus-v3/dist/index.js
index 2164443..45c4e4e 100644
--- a/node_modules/@orama/plugin-docusaurus-v3/dist/index.js
+++ b/node_modules/@orama/plugin-docusaurus-v3/dist/index.js
@@ -34,7 +34,7 @@ function OramaPluginDocusaurus(ctx, options) {
         },
         async allContentLoaded({ actions, allContent }) {
             var _a, _b;
-            const isDevelopment = process.env.NODE_ENV === 'development' || !((_a = options.cloud) === null || _a === void 0 ? void 0 : _a.oramaCloudAPIKey);
+            const isDevelopment = process.env.NODE_ENV === 'development' || (options.cloud && !options.cloud.oramaCloudAPIKey);
             const docsInstances = [];
             const oramaCloudAPIKey = (_b = options.cloud) === null || _b === void 0 ? void 0 : _b.oramaCloudAPIKey;
             const searchDataConfig = [
diff --git a/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/useOrama.js b/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/useOrama.js
index c975c72..14dafa0 100644
--- a/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/useOrama.js
+++ b/node_modules/@orama/plugin-docusaurus-v3/dist/theme/SearchBar/useOrama.js
@@ -24,7 +24,7 @@ const useOrama = () => {
     const isBrowser = (0, useIsBrowser_1.default)();
     (0, react_1.useEffect)(() => {
         async function loadOrama() {
-            if (endpoint) {
+            if (endpoint && endpoint.url) {
                 setSearchBoxConfig({
                     oramaInstance: new client_1.OramaClient({
                         endpoint: endpoint.url,
